inc/functions_post.php

```diff
@@ -401,64 +401,81 @@ function build_postbit($post, $post_type=0)
 		}
 
 		// Display profile fields on posts - only if field is filled in
-		if(is_array($profile_fields))
```

Comment:
@JN-Jones I'm against this:
https://github.com/mybb/mybb/pull/1952/files#diff-4139bceb91b5897c2a9cddc06430deacR416

The `editableby` field should only affect edition and not display of the field.

Also, what I noted before that hasn't been updated in the PR. The code shouldn't modify the `showthread.php` file and assume that only the thread postbit should display profile fields if using the `build_postbit` function.

New issues for discussion about those changes can be created, I'm not against discussion but instead out-of-the-box changes that can affect more that what the issue at hand is meant to take.